common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -857,6 +859,8 @@
   "feature.wynntils.partyMembersOverlay.overlay.partyMembers.maxPartyMembers.description": "How many party members should be shown?",
   "feature.wynntils.partyMembersOverlay.overlay.partyMembers.maxPartyMembers.name": "Max Party Members",
   "feature.wynntils.partyMembersOverlay.overlay.partyMembers.name": "Party Members",
+  "feature.wynntils.perClassGuildContribution.description": "Remembers what contribution XP%% you set per class and applies when you select that class.",
```

Comment:
xp%%?

Reply:
That's how you get it to show %, using just one says format error
![image](https://github.com/Wynntils/Artemis/assets/8337467/67a1341d-62c2-4054-b2d6-b064c91796f2)

Reply:
`\%`

Reply:
That doesn't work either. The other %'s use %% like health, mana, mem_pct etc

Reply:
Yeah but.. "Format error:"?

Reply:
Just how I18n does it I guess
```
public static String get(String translateKey, Object... parameters) {
        String string = language.getOrDefault(translateKey);

        try {
            return String.format(string, parameters);
        } catch (IllegalFormatException var4) {
            return "Format error: " + string;
        }
    }
```

Reply:
%% should be the way.. weird that it doesnt work?

Reply:
%% does work. It's a single and \% that doesn't.